test/e2e/v2/e2e_v2_test.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
> test/e2e/v2/e2e_v2_test.go

Same comment regarding the path, drop the latter `v2` in the path.

Reply:
👍